kvm.rb

```diff
@@ -2,9 +2,14 @@
 
 class Kvm < Formula
   homepage "https://www.github.com/aspnet/Home"
-  version "1.0.0-beta1"
+  version "1.0.0-beta3"
```

Comment:
Yeah, I think that what I will actually do is change the URL to point at a specific tag for the current version (which doesn't yet exist on aspnet/Home) and use "HEAD" to refer to the tip of the 'dev' branch. We'll have to keep updating the formula, but that's a) pretty easy, and b) basically what [others do](https://github.com/Homebrew/homebrew/blob/master/Library/Formula/rbenv.rb) (though they often build release packages in `.tar.gz` form)